code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -385,19 +390,42 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/chair/stool/bar, 0)
 /obj/item/chair/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK, damage_type = BRUTE)
 	if(attack_type == UNARMED_ATTACK && prob(hit_reaction_chance) || attack_type == LEAP_ATTACK && prob(hit_reaction_chance))
 		owner.visible_message(span_danger("[owner] fends off [attack_text] with [src]!"))
+		if(take_chair_damage(damage, damage_type, MELEE)) // Our chair takes our incoming damage for us, which can result in it smashing.
+			smash(owner)
 		return TRUE
 	return FALSE
 
 /obj/item/chair/afterattack(atom/target, mob/user, click_parameters)
-	if(!prob(break_chance))
+	if(!ishuman(target))
+		return
+
+	var/mob/living/carbon/human/give_this_fucker_the_chair = target
+
+	// Here we determine if our attack is against a vulnerable target
+	var/vulnerable_hit = (check_behind(user, give_this_fucker_the_chair) || give_this_fucker_the_chair.get_timed_status_effect_duration(/datum/status_effect/staggered))
```

Comment:
Probably could check for lying down instead of staggered and move staggered to the if check? (vulnerable_hit || staggered)